cli.mjs

```diff
@@ -185,22 +242,23 @@ async function deploymentPrompt() {
 }
 
 /** Prompt the user to enter the ID of a study */
+// helper to check if the given study (input) is in firestore
+async function validateStudyFirebase(input) {
+  // subcollection is programmatically generated, if it doesn't exist then input must not be a valid studyID
+  const studyIDCollections = await getStudyRef(input).listCollections();
+  return studyIDCollections.find((c) => c.id === PARTICIPANTS_COL);
+}
```

Comment:
Can you move this helper function (along with `validateParticipantFirebase`) into a new section for validation functions? It looks like some of the comments aren't above the correct functions anymore